### ISSUE TITLE
svrx无法启动

### ISSUE THREAD

**Issue by Assone** [svrxjs/svrx#191]
**Describe the bug**
svrx无法启动并报错

**To Reproduce**
1. 打开终端
2. 输入svrx
3. 出现报错`[error] Error: install error: package '@svrx/svrx' not found: must call as: npm.load(callback)`

 - OS: macOS 11.0.1
 - Node: v15.2.1
 - Browser: chrome

**Comment by xuchaoying** [comment]
svrx 安装的方式是怎么样的呢？ 安装过程有没有报错？

**Comment by Assone** [comment]
> svrx 安装的方式是怎么样的呢？ 安装过程有没有报错？

`yarn global add @svrx/cli`或者`npm install -g @svrx/cli`安装的，安装的时候没有报错，就启动的时候会报这个错误

**Comment by arihantverma** [comment]
facing the same issue while trying to run this eleventy starter — https://github.com/brycewray/eleventy_solo_starter_njk. Installed `@svrx/cli` globally like @Assone did.

**Comment by xuchaoying** [comment]
It seems a bug of `npm@7`. 

In the version of 6 and former versions, `npm.load()` accepts two arguments: cli options and callback, but now at 7 it only accepts one argument as callback, see [the code](https://github.com/npm/cli/blob/latest/lib/npm.js#L128).

While its [changelog](https://github.com/npm/cli/blob/latest/CHANGELOG.md#programmatic-usage) says that

> `npm.load()` MUST be called with two arguments (the parsed cli options and a callback).

You can try `npm@6` temporarily. And we'll try to abandon `npm.load()` very soon. @Assone @arihantverma

**Comment by xuchaoying** [comment]
Fixed on `@svrx/svrx@1.1.6`

**Comment by Assone** [comment]
it still not work

```
[error] Error: install failed: Error: Command failed:
      npm install --prefix=/var/folders/nb/kty93txs6jx4lwvdr6vbqt8h0000gn/T/tmp-457087uAbCdf7MVOc @svrx/svrx@1.1.6 -g

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npm.taobao.org/@babel/highlight/-/highlight-7.13.10.tgz - [not_found] document not found
npm ERR! 404
npm ERR! 404  '@babel/highlight@https://registry.npm.taobao.org/@babel/highlight/-/highlight-7.13.10.tgz' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/reset0/.npm/_logs/2021-03-26T15_11_16_784Z-debug.log
```
